src/components/ProductPage/components/specifications/styles.module.scss

```
@@ -4,14 +4,13 @@
 
 
 
-
+.container {
+    @include flex (center, center, column);
```

```suggestion
    @include flex(center, center, column);
```

Guarda con los espacios innecesarios

---

done